src/components/line-chart-visualization/line-chart-visualization.component.scss

```diff
@@ -0,0 +1,7 @@
+:host {
+  ::ng-deep {
```

Comment:
What about just adding a container element in the line-chart component and setting a max-width on that?